edalize/templates/apicula/apicula-makefile.j2

```diff
@@ -0,0 +1,23 @@
+#Auto generated by Edalize
+
+TARGET   := {{ name }}
+CST_FILE := {{ cst_file }}
+NEXTPNR_OPTIONS     := {{ nextpnr_options|join(' ') }}
+DEVICE := {{ device|join('') }}
```

Comment:
In the .core I forgot to put device option between square brackets so maybe no need to add |join(''). I will wait your review to modify this and other things if needed.

Reply:
If the device option is moved to `members` as noted above, then I believe we can also drop this join